docs/book/src/troubleshooting.md

```diff
@@ -71,6 +71,9 @@ curl ${SERVICE_ACCOUNT_ISSUER}/.well-known/openid-configuration
 curl ${SERVICE_ACCOUNT_ISSUER}/openid/v1/jwks
 ```
 
+In a managed AKS cluster you also can get this OIDC issuer endpoint from the cluster json representation in ``properties.oidcIssuerProfile.issuerURL`` (you have to set API version to at least 2022-09-01). To test the endpoint try to access ``${AKS_oidcIssuerProfile_issuerURL}/.well-known/openid-configuration`` and you should get the OIDC configuration. If you can an error your OIDC endpoint deployment was failing.
```

Comment:
yes, but this requires a pod in which you can jump. If you mostly have distroless containers this is hard to debug.
Using the endpoint url from ``properties.oidcIssuerProfile.issuerURL`` is much faster then trying to jump and to decode the JWT token.

It's very important to tell the Azure support that this url is broken and nothing inside our containers, otherwise they are searching the issue on customer side and not inside the AKS service.